open-sphere-base/mantle/src/main/java/io/opensphere/mantle/icon/impl/gui/IconManagerFrame.java

```diff
@@ -322,7 +333,7 @@ private void createMenuBarFileMenuItems()
      */
     private void createTreePopupMenuItems(JPopupMenu puMenu)

```

Comment:
I see both createIconPopupMenuItems & createMenuBarEditItems() use the same JMenuItems. You guys could just create 1 of those and reuse it.  for example ->

`JMenuItem deleteSelectedMI = new JMenuItem("Delete Selected");
        deleteSelectedMI.addActionListener(e -> deleteSelected());
        puMenu.add(deleteSelectedMI);`
just make 1 of deleteSelectedMI and reuse it

Reply:
The only difference between the two is delete selected icon set so that can get stay the same probably

Reply:
addressed